packages/tools/webpack-fluid-loader/src/loader.ts

```diff
@@ -66,6 +66,8 @@ export interface IRouterliciousRouteOptions extends IBaseRouteOptions {
 
 export interface ITinyliciousRouteOptions extends IBaseRouteOptions {
     mode: "tinylicious";
+    tenantId?: string;
+    tenantSecret?: string;
```

Comment:
I would rather not add these here. You can instead directly provide the tenantId and secret to the InsecureTokenProvider for tinylicious.

Reply:
Ditto

Reply:
Long term, we don't want these as part of options. If you are planning to work on the long term solution soon, then its fine to keep it here :-).
Otherwise, if someone starts using these options, it will just become harder to remove.